benches/src/erc20.rs

```diff
@@ -0,0 +1,194 @@
+use alloy::{
+    network::{AnyNetwork, EthereumWallet},
+    primitives::Address,
+    providers::ProviderBuilder,
+    sol,
+    sol_types::SolConstructor,
+    uint,
+};
+use alloy_primitives::U256;
+use e2e::{receipt, Account};
+use koba::config::Deploy;
+
+use crate::ArbOtherFields;
+
+const RPC_URL: &str = "http://localhost:8547";
+
+sol!(
```

Comment:
Is there any way to reuse `abi.rs` from integration tests here?

Reply:
This would couple benches with e2e tests, so it's better not to do this.

Reply:
Fair enough.